tests/airport.spec.js

```diff
@@ -150,12 +150,11 @@ console.log(`Test 6 - Is Airport full?`);
 
 //Arrange
 
-airport = new Airport();
-expected = 10;
+airport = new Airport([new Plane()], 1);
```

Comment:
I refactored the test and added a constructor so it was easier to manipulate the variable values in the test. This then allowed me to change the isAirportFull method as before it was just telling me that it had a capacity of 10.